spec/features/card_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+require 'card'
+
+describe "A card" do
+  it "should have a suit" do
+    QueenOfHearts = Card.new('12', 'Hearts')
+    QueenOfHearts.suit.wont_be_empty
```

Comment:
Capitalized words in ruby are `Constants`. Keep your local vars to `lower_snake_case`.